src/org/zaproxy/zap/extension/websocket/db/TableWebSocket.java

```diff
@@ -583,6 +583,7 @@ public void insertOrUpdateChannel(WebSocketChannelDTO channel) throws DatabaseEx
 			
 					stmt.setString(1, channel.host);
 					stmt.setInt(2, channel.port);
+					logger.debug("url (length " + channel.url.length() + "):" + channel.url);
```

Comment:
Consider checking if debug is enabled to avoid building the string when not needed (which is most of the cases, debug is not the default).